custom_components/midea_ac_lan/climate.py

```diff
@@ -417,14 +417,12 @@ def current_temperature(self) -> float:
 
 
 class MideaC3Climate(MideaClimate):
-    def __init__(self, device: Any, entity_key: str, zone: str) -> None:
+    def __init__(self, device: Any, entity_key: str, zone: int) -> None:
         super().__init__(device, entity_key)
         self._zone = zone
         self._modes = [HVACMode.OFF, HVACMode.AUTO, HVACMode.COOL, HVACMode.HEAT]
         self._power_attr = (
-            C3Attributes.zone1_power
-            if zone == "zone1_power"
-            else C3Attributes.zone2_power
+            C3Attributes.zone1_power if zone == 0 else C3Attributes.zone2_power
```

Comment:
Please full revert my change to the original code

Reply:
ok